.drone.yml

```diff
@@ -108,6 +114,8 @@ matrix:
       RICHDOCUMENTS_CHECKSUM: dbca97f24abecb10a23686361e56a1c91c24d9007a5c142169c75a836b8cebed
       LDAP_VERSION: v0.11.0
       LDAP_CHECKSUM: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+      ONLYOFFICE_VERSION: v1.3.0
```

Comment:
Let's add this also for the 10.0.8 matrix permutation, then we don't need to fight with conditions.

Reply:
done